demo/mobile/package.json

```diff
@@ -14,18 +14,18 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "@raincatcher-examples/angularjs-extensions": "1.1.1",
-    "@raincatcher-examples/step-accident": "1.1.1",
-    "@raincatcher-examples/step-vehicle-inspection": "1.1.1",
-    "@raincatcher/angularjs-auth": "1.1.1",
-    "@raincatcher/angularjs-auth-keycloak": "1.1.1",
-    "@raincatcher/angularjs-auth-passport": "1.1.1",
-    "@raincatcher/angularjs-workflow": "1.1.1",
-    "@raincatcher/angularjs-workorder": "1.1.1",
+    "@raincatcher-examples/angularjs-extensions": "1.1.2",
+    "@raincatcher-examples/step-accident": "1.1.2",
+    "@raincatcher-examples/step-vehicle-inspection": "1.1.2",
+    "@raincatcher/angularjs-auth": "1.1.2",
+    "@raincatcher/angularjs-auth-keycloak": "1.1.2",
+    "@raincatcher/angularjs-auth-passport": "1.1.2",
+    "@raincatcher/angularjs-workflow": "1.1.2",
+    "@raincatcher/angularjs-workorder": "1.1.2",
     "@raincatcher/datasync-client": "1.1.1",
```

Comment:
datasync-client, logger and wfm were also updated to `1.1.2`. Is this missing a version bump?

Reply:
Thanks for the catch, was just reviewing this locally, failed to pick the version bumps on a new OS i'm in because of the symlink